packages/common-all/src/analytics.ts

```diff
@@ -154,6 +154,7 @@ export enum AppNames {
 
 export enum GraphEvents {
   GraphThemeChanged = "GraphThemeChanged",
+  LocalGraphSidePanel = "LocalGraphSidePanel",
```

Comment:
Can we follow the noun + verb convention?  Given that you've named it `DendronGraphPanel.tsx`, let's keep that terminology - so how about `GraphPanelUsed` for this event name? 

We can start calling this GraphPanel (side) vs GraphView (full page webview)

Reply:
updated event and changed `Local Graph View` to `Graph Panel`